src/components/Intro.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import Button from './Button.js';
+import bus from '../assets/bus.png';
+
+const Intro = () => {
+  return (
+    <div className="flex flex-col items-center xl:flex-row">
+      <div>
+        <h1 className="flex flex-col text-3xl xl:text-5xl font-raleway font-extrabold mx-12 xl:mx-28 mt-12 xl:mt-32">
```

Comment:
On my side the text part is not rendering correctly. The image seem to take most of the place. 
Consider using `grid grid-cols-2` if flex isn't rendering correctly. Attached screenshot for reference.

With grid:
![image](https://user-images.githubusercontent.com/28519395/155215570-aa71eaf5-61e9-43db-aef9-d785da25ec2a.png)

With Flex:
![image](https://user-images.githubusercontent.com/28519395/155215664-539d107d-5021-4a40-a021-d397a4dc529f.png)